games/Game.spec.js

```diff
@@ -1,6 +1,6 @@
 const mongoose = require('mongoose');
 
-const Game = require('./Game');
+
```

Comment:
I would suggest importing the Game mongoose model in here to be able to test against it.